### PR TITLE
2401

### PR DESCRIPTION
**Pull request by Me1oyy**
Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,13 @@ In this section, we provide a detailed performance analysis of FinMA compared to
 | BigData22             | stock movement prediction        | 7,164  | tweets, historical prices | text, time series | Public          | [13]  |
 | ACL18                 | stock movement prediction        | 27,053 | tweets, historical prices | text, time series | MIT License     | [14]  |
 | CIKM18                | stock movement prediction        | 4,967  | tweets, historical prices | text, time series | Public          | [15]  |
+| Lending Club          | credit scoring                   | 1,3453 | financial information     | table             | CC0 1.0         | [16]  |
```
Comment (jiminHuang) [inline README.md:131]: Have these tasks been added to the flare.py?

**Review by jiminHuang — CHANGES_REQUESTED** [review]

**Comment by Me1oyy** [comment]
tasks have been added to flare.py and __init__.py

File: src/tasks/__init__.py [hunk 2]
```diff
@@ -50,6 +50,14 @@
     "flare_zh_cikm": flare.ZHCIKM,
     "flare_zh_finqa": flare.ZHFinQAE,
     "flare_zh_convfinqa": flare.ZHConvFinQA,
+    "lengdingclub": flare.lendingclub,
```
Comment (jiminHuang) [inline src/tasks/__init__.py:53]: Please follow our task naming rules, formatting as "flare_"

**Review by jiminHuang — CHANGES_REQUESTED** [review]

**Comment by Me1oyy** [comment]
tasks names changed

**Pull request closed without merging**